modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/JBossProductType.java

```diff
@@ -143,6 +144,8 @@ private static JBossProductType determineJBossProductTypeViaProductConfFile(File
                 productType = JBossProductType.BPMS;
             } else if (slot.equals("dv")) {
                 productType = JBossProductType.JDV;
+            } else if (slot.equals("wildfly-full")) {
```

Comment:
I have had that exception before and the monitoring hasn't started, but now it's working fine on a RHQ 4.13, without any messages.